lldb/include/lldb/Target/Process.h

```diff
@@ -1571,6 +1571,25 @@ class Process : public std::enable_shared_from_this<Process>,
   virtual size_t ReadMemory(lldb::addr_t vm_addr, void *buf, size_t size,
                             Status &error);
 
+  /// Read from multiple memory ranges and write the results into buffer.
```

Comment:
Where should we note that this is something to be overridden? Or do they override something else at a lower level and that's already noted?